src/main/java/mate/academy/rickandmorty/repository/CharacterRepository.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.rickandmorty.repository;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.Character;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CharacterRepository extends JpaRepository<Character, Long>,
+        JpaSpecificationExecutor<Character> {
+    List<Character> findByNameContaining(String name);
+
+    @Query("from Character ORDER BY RAND() limit 1")
```

Comment:
```suggestion
    @Query("FROM Character ORDER BY RAND() LIMIT 1")
```
consistency is key, if you write some SQL keywords as uppercase - all of them should be uppercased